lib/bolt/transport/winrm.rb

```diff
@@ -162,7 +162,11 @@ def run_task(target, task, arguments, _options = {})
                 interpreter = select_interpreter(remote_task_path, target.options['interpreters'])
                 if interpreter
                   path = interpreter
-                  args = [remote_task_path]
+                  # Quote path if not quoted
+                  unless path =~ /^'.*'$/ || path =~ /^".*"$/
+                    path = "\"#{path}\""
```

Comment:
Should we prefer single quotes here?

Reply:
Based on #1231 it seems like yes, since escaping characters in single quotes seems easier (based on reading some StackOverflow posts). @Iristyle thoughts?

Reply:
I was thinking the place to change this would be `Powershell#execute_process`, which is what currently handles escaping the arguments. So we ought to change it to just do the same thing to the command.